### PR TITLE
Working updateName and updateEmail functions

### PR DESCRIPTION
**Pull request by conniec218**
Working updateName and updateEmail functions

### CODE REVIEW

File: api/models/User.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ let User;
 // User Schema Definition
 const userSchema = new mongoose.Schema({
 	name: { type: String, required: true },
+	email: { type: String, default: '', required: true },
```
Comment (shannonphu) [inline api/models/User.js]: I don't think there needs to be a default.

File: api/index.js [hunk 2]
```diff
@@ -35,6 +35,13 @@ app.use((err, req, res, next) => {
 	res.status(500).json({ err: err.message });
 });
 
+User.create('Joe Bruin', 'email', 'bleh17')
```
Comment (shannonphu) [inline api/index.js]: Remove this code from the PR that just tests your methods.

File: docker-compose.yml [hunk 3]
```diff
@@ -12,9 +12,9 @@ services:
...
-      - "3000:3000"
-    links:
-      - api
\ No newline at end of file
+  # web_client:
+  #   build: ./client
+  #   ports:
+  #     - "3000:3000"
+  #   links:
+  #     - api
```
Comment (TimothyGu) [inline docker-compose.yml]: Stray change.

File: api/index.js [hunk 4]
```diff
@@ -35,6 +35,13 @@ app.use((err, req, res, next) => {
...
+	.then(user => {
+		console.log(user);
+		return user.updateEmail('no');
+	})
+	.catch(error => console.error(error));
+
```
Comment (TimothyGu) [inline api/index.js]: Stray change.

“No.”

File: api/models/User.js [hunk 5]
```diff
@@ -234,6 +237,46 @@ userSchema.methods.removeSkill = function (skill) {
...
+/*
+ * Update name for the User object
+ * @param {string} name to update the User object
+ * @returns {User} changed user object
+ */
+
+userSchema.methods.updateName = function (name) {
+	this.name = name;
+
+	return new Promise((resolve, reject) => {
```
Comment (shannonphu) [inline api/models/User.js]: Both methods repeat this code that saves the currently updated object. Maybe we can move this code to a separate method?

File: docker-compose.yml [hunk 6]
```diff
@@ -12,9 +12,9 @@ services:
...
     links:
       - db_mongo
-  web_client:
-    build: ./client
-    ports:
-      - "3000:3000"
-    links:
-      - api
\ No newline at end of file
+  # web_client:
```
Comment (shannonphu) [inline docker-compose.yml]: Don't include this commented out code in the PR.

**Comment by TimothyGu** [comment]
Next time, could you use a more descriptive pull request title? Use what you have in the description as the title.

File: api/models/User.js (hunk 5 above)
Comment (TimothyGu) [reply api/models/User.js]: In fact, half of all methods in the file has this chunk of code.

Comment (shannonphu) [reply api/models/User.js]: That’s true, we’ll fix this this a separate PR. So this can be left here for now.

Comment (TimothyGu) [reply api/models/User.js]: (#74)

File: api/index.js [hunk 7]
```diff
@@ -7,9 +7,6 @@ const server = http.createServer(app);
 const cors = require('cors');
 const config = require('./config');
 
-// Connect to database
-require('./models');
-
```
Comment (TimothyGu) [inline api/index.js]: Stray change.

File: api/index.js [hunk 8]
```diff
@@ -7,7 +7,6 @@ const server = http.createServer(app);
 const cors = require('cors');
 const config = require('./config');
 
-// Connect to database
```
Comment (TimothyGu) [inline api/index.js]: Stray change

**Comment by TimothyGu** [comment]
For future reference, this takes care of the first two items of https://github.com/uclaacm/mentorq/issues/46.

**Pull request merged**